docs/docsite/rst/community/index.rst

```diff
@@ -8,8 +8,72 @@ Welcome to the Ansible Community Guide!
 
 The purpose of this guide is to teach you everything you need to know about being a contributing member of the Ansible community.
 
+All types of contributions are welcome, and necessary to Ansible's continued success.
+
 To get started, please read and understand the :ref:`code_of_conduct`, and then select one of the following topics.
 
+
+Getting Started
+===============
+
+* Read the :ref:`official documentation <ansible_documentation>`
```

Comment:
I'd skip this. If they're reading this page, they're already looking at the documentation, and they have lots of ways to get to other bits of it. Plus, but putting "read the official documentation" first, we're suggesting that you need to "read the book" before you can do anything else . . . which is not the message we want to send.

Reply:
Good point, removed